commit 1670c7af02ef37580aa286ff8561e7beee6381a8 (refs/changes/00/2500/2)
Author: Robert Mustacchi <rm@joyent.com>
Date:   2017-09-13T21:21:44+00:00 (2 years, 1 month ago)
    
    OS-6324 Need support for LSI 9305-24i
    Reviewed by: Dan McDonald <danmcd@joyent.com>
    Approved by: Joshua M. Clulow <jmc@joyent.com>

diff --git a/overlay/generic/etc/driver_aliases b/overlay/generic/etc/driver_aliases
index eda1c7b9..8959edc1 100644
--- a/overlay/generic/etc/driver_aliases
+++ b/overlay/generic/etc/driver_aliases
@@ -920,6 +920,7 @@ mpt_sas "pciex1000,94"
 mpt_sas "pciex1000,95"
 mpt_sas "pciex1000,96"
 mpt_sas "pciex1000,97"
+mpt_sas "pciex1000,c4"
 mr_sas "pciex1000,52"
 mr_sas "pciex1000,53"
 mr_sas "pciex1000,5b"
